docs/event.md

```diff
@@ -43,6 +43,43 @@ is fired. The following key values are also supported:
  - `%arguments` - The arguments passed when the event was dispatched/triggered.
 
 
+@signature `(KEY_LOOKUP EVENT)='CALL_EXPRESSION'`
+
+Listens to an event in the [can-stache/expressions/key-lookup specified scope] and calls the [can-stache/expressions/call] when that event occurs.
+
+```
+<div 
+  ($click)="./activityrecorder.recordClick()"
+  (./activityrecorder clickrecorded)="doSomething()"
+/>
+```
+In this example, `activityrecorder` exists on the current scope and is defined as:
+```
+var ActivityRecorder = DefineMap.extend({
+	recordClick: function(){
+		this.dispatch("clickrecorded");
+	}
+});
+
+var scope = new DefineMap({
+	activityrecorder: new ActivityRecorder()
+})
+```
+
+@param {can-stache/expressions/key-lookup} KEY_LOOKUP A key to specify the context for the `EVENT`.
```

Comment:
A key to specify the context for the `EVENT` binding. The object specified by `key` will have its event named `EVENT` bound.